Taskfile.yml

```diff
@@ -85,13 +85,13 @@ tasks:
     desc: "[re]generate all generated files"
     cmds:
       - task: mod:tidy
-      - task: charts:generate
       # update-licenses may update licenses/boilerplate.go.txt which is used
       # for _some_ of k8s:generate. For simplicity, we just run update-licenses
       # twice.
       - task: dev:update-licenses
       - task: k8s:generate
       - task: dev:update-licenses
+      - task: charts:generate
```

Comment:
I'm just curious why charts generate needs to be moved down the stack

Reply:
Now I get why we need to perform `k8s:generate` before making sync of our generated roles.